specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json

```diff
@@ -5034,9 +5034,9 @@
         },
         "userAssignedIdentities": {
           "type": "object",
-          "x-ms-client-name": "IdentityUserAssignedIdentitiesValue",
           "additionalProperties": {
             "type": "object",
+            "x-ms-client-name": "IdentityUserAssignedIdentitiesValue",
```

Comment:
Is this required to define within `additionalProperties`? I don't see it from the [extension document](http://azure.github.io/autorest/extensions/#x-ms-client-name).

Reply:
The reason why I added **client-name** is that **autorest-v3** does not know how to name this **additionalProperties**. It once generated a random name: **Componentssgqdofschemasidentitypropertiesuserassignedidentitiesadditionalproperties**, which is inconsistent with the name **IdentityUserAssignedIdentitiesValue** autorest-v2 generated. The details can be found here: https://github.com/Azure/autorest.csharp/issues/614
My last pr incorrectly put it in an upper level, and I found this issue in dotnet sdk; 
Moved it to the inner scope, re-verified it worked correctly.